.cargo/config.toml

```diff
@@ -9,3 +9,8 @@ rustflags = [
   "-C", "link-arg=-undefined",
   "-C", "link-arg=dynamic_lookup",
 ]
+
+[target.x86_64-unknown-linux-gnu]
+rustflags = [
+    "-C", "link-arg=-undefined"
+]
```

Comment:
Do we want to include these flags for all builds? Interested in if there's a way to set these globally and then have the OS X builds override with their additional build flags

Reply:
We can remove this now. This was added as part of an experiment I was performing to fix linux based cargo test builds with certain versions of GCC and I failed to remove it once I had performed that test.